packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -202,9 +238,163 @@ export default async function initSanity(
   // Ensure we are using the output path provided by user
   outputPath = answers.outputPath
 
+  if (initNext) {
+    const useTypeScript = await promptForTypeScript(prompt)
+
+    const fileEnding = useTypeScript ? 'ts' : 'js'
+
+    const embeddedStudio = await promptForEmbeddedStudio(prompt)
+
+    if (embeddedStudio) {
+      const useAppDir = await promptForAppDir(prompt)
+
+      // find source path (app or pages dir)
+      const srcDir = useAppDir ? 'app' : 'pages'
+      let srcPath = path.join(workDir, srcDir)
+
+      if (!existsSync(srcPath)) {
+        srcPath = path.join(workDir, 'src', srcDir)
+        if (!existsSync(srcPath)) {
+          await fs
+            .mkdir(srcPath, {recursive: true})
+            .catch(() => debug('Error creating folder %s', srcPath))
```

Comment:
in the case it fails, it's _most_ likely because the path already exists (ENOENT), in which case it should be totally fine to move forward. it's the same logic used on env creation (to check if folder exists before writing file)